ui/pages/confirmations/components/confirm/info/approve/edit-spending-cap-modal/edit-spending-cap-modal.tsx

```diff
@@ -116,10 +128,17 @@ export const EditSpendingCapModal = ({
     setCustomSpendingCapInputValue(formattedSpendingCap.toString());
   }, [customSpendingCapInputValue, formattedSpendingCap]);
 
+  const showDecimalError = useMemo(() => {
```

Comment:
Minor, do we need a memo for this as some basic math will have negligible performance cost?